src/classes/EP_Task_UTIL.cls

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
* This method is empty, I suggest to remove it.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.